pkg/datapath/connector/add.go

```diff
@@ -45,6 +45,6 @@ func truncateString(epID string, maxLen uint) string {
 }
 
 // DisableRpFilter tries to disable rpfilter on specified interface
-func DisableRpFilter(ifName string) error {
+func DisableRpFilter(sysctl sysctl.Sysctl, ifName string) error {
```

Comment:
I'm not sure I have actionable feedback here, but there is out of tree code multi-network code which calls this function from the CNI binary. In the end, I guess it is the responsibility of downstream to catch up which those kind of changes.

Reply:
If this function is indeed used externally then they will need to change it, but that shouldn't be hard with the `NewDirectSysctl` constructor I added.